rust/trace/src/sys_pid.rs

```diff
@@ -0,0 +1,30 @@
+use libc;
```

Comment:
licence

Reply:
Bah.  Is there some kind of way to set a preference in git/vim so that new files have this automatically filled in?  Or at least a commit hook to enforce it?

Reply:
Good Q, I don't know anything offhand but i'm sure it exists.